content/departments/engineering/dev/process/principles-and-practices.md

```diff
@@ -29,3 +29,46 @@ If you have an idea for something you want to work on, then you have a few optio
+- It's up to _you_ to take a step back when you realize that you're just building something because it's fun to build, or because it's new & shiny, but not because it has value to customers.
+- It's up to _you_ to do everything you can as an engineer to provide as much value as possible to our customers.
+
+Sometimes that's writing docs, sometimes it's getting deep into the weeds and doing performance optimizations for weeks, sometimes you should jump on a call with a customer to find out exactly what they want or what's bugging them, sometimes you need to record a demo video to make sure what you built actually lands with customers.
+
+Sometimes it means helping out other teams, because they're completely under water, but if they don't manage to pull this off, then your customers are impacted.
+
+Sometimes it means changing code that you don't officially own on paper – because you need the code to be changed in order to fix your customer's problem. You need team A to build FooWidget but team A is busy with something else and said they can't do it at the moment? Forget about the which-teams-owns-which-folder level of ownership and start asking: can _you_ build X? Can you build a prototype and get sign-off from team A and then continue, because your customer is waiting for it? You can expect your collaborators to be open towards contributions, so why not make use of that?
+
+Sometimes it's about civil disobedience. That doesn't mean you should break the law or behave against our values. No, on the contrary. It means that sometimes a process or a convention has outlived its usefulness (or goes against our values) and the only way to demonstrate that is by not doing it, and calling it out even if no one asked you. Other times it means breaking the chain of command, if, for example, you think something's going to happen that will affect a customer in a negative way but you don't think anyone's listening.
+
+True ownership revolves around one thing: "what can I do – with my skills, my expertise, my experience – to provide value to my customers?"
+
+All of that responsibility is daunting because, of course, it's much more comfortable to lean back and say "I'll just work on my tickets, thank you." But that's not what you were hired for.
+
+You need to embrace high agency and customer focus and the responsibility that comes with it. Because what you built, the problems you solve, the value you provide – it's yours. You can be proud of it. You get to demo it, you get to share it, you get to listen to the customer say "this is fantastic! this saves us so much time!", you get to say "yes, I built _that_". Once you get a taste of that, there's no going back.
```

Comment:
```suggestion
You need to embrace high agency and customer focus and the responsibility that comes with it. Because what you built, the problems you solve, the value you provide – it's yours. You can be proud of it. You get to demo it, you get to share it, you get to listen to the customer say "this is fantastic! this saves us so much time!", you get to say "yes, I built _that_". Once you get a taste of it, there's no going back.
```

Or something; I think using "that" so much in the previous sentence makes this occurrence of "that" a bit confusing. Could also use "a taste of this" or even just "a taste".